src/main/kotlin/com/github/mitchellolsthoorn/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -0,0 +1,59 @@
+package com.github.mitchellolsthoorn.testgenie.services
+
+import com.intellij.ide.highlighter.JavaFileType
+import com.intellij.openapi.project.Project
+import com.intellij.psi.JavaCodeFragmentFactory
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.ui.EditorTextField
+import com.intellij.ui.components.JBScrollPane
+import java.awt.BorderLayout
+import java.awt.Dimension
+import javax.swing.*
+
+class TestCaseDisplayService(private val project: Project) {
+
+    val mainPanel: JPanel = JPanel()
+    private val applyButton: JButton = JButton("Apply")
+    private val allTestCasePanel: JPanel = JPanel()
+    private val scrollPane: JBScrollPane = JBScrollPane(allTestCasePanel)
+
+    init {
+        allTestCasePanel.layout = BoxLayout(allTestCasePanel, BoxLayout.Y_AXIS)
+        mainPanel.layout = BorderLayout()
+        mainPanel.add(applyButton, BorderLayout.SOUTH)
```

Comment:
I like your big "apply" button, but should we be consistent with the size of our buttons across the different sections of the tool window?

Reply:
I made it bigger so that it's easier to find and aligns better with the rest of that UI. Maybe we could change it later if it is indeed inconsistent.